src/test/java/cloud/eppo/helpers/JacksonJsonDeserializer.java

```diff
@@ -0,0 +1,92 @@
+package cloud.eppo.helpers;
+
+import cloud.eppo.Utils;
+import cloud.eppo.api.EppoValue;
+import cloud.eppo.exception.JsonParsingException;
+import cloud.eppo.helpers.dto.adapters.EppoModule;
+import cloud.eppo.ufc.dto.BanditParametersResponse;
+import cloud.eppo.ufc.dto.FlagConfigResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
Cool this is in the `test` subdirectory so theoretically it won't be bundled with builds its not used 👌